devices.js

```diff
@@ -2216,6 +2216,28 @@ const devices = [
         description: 'Tint LED bulb GU10/E14/E27 350/470/806 lumen, dimmable, opal white',
         extend: generic.light_onoff_brightness_colortemp,
     },
+
+    // Salus
+    {
+        zigbeeModel: ['SP600'],
+        model: 'SP600',
+        vendor: 'Salus',
+        description: 'Smart plug',
+        supports: 'on/off, power measurement',
+        fromZigbee: [fz.generic_state, fz.ignore_onoff_change, fz.EDP_power, fz.ignore_metering_change],
```

Comment:
Could you rename the `EDP_power` converter to `generic_power`?

Reply:
sure, no problem.